docs/externalnode.md

```diff
@@ -0,0 +1,309 @@
+# ExternalNode
+
+## Table of Contents
+<!-- toc -->
+- [What is ExternalNode?](#what-is-externalnode)
+- [Prerequisites](#prerequisites)
+- [The ExternalNode resource](#the-externalnode-resource)
+  - [Interfaces](#interfaces)
+- [Apply Antrea NetworkPolicy to ExternalNode](#apply-antrea-networkpolicy-to-externalnode)
+- [Traffic model](#traffic-model)
+  - [OVS pair ports](#ovs-pair-ports)
+  - [OpenFlow pipeline](#openflow-pipeline)
+    - [Non-IP packet](#non-ip-packet)
+    - [IP packet](#ip-packet)
+- [RBAC](#rbac)
+- [Limitations](#limitations)
+<!-- /toc -->
+
+## What is ExternalNode?
+
+`ExternalNode` is a CRD API that manages the network connectivity and security
+on a virtual machine or a bare-metal server which is not a Kubernetes Node.
+It supports specifying which network interfaces on the external Node are expected
+to be guarded with Antrea NetworkPolicy rules. The virtual machine or bare-metal
+server presented by `ExternalNode` can be either Linux or Windows. "external Node"
+will be used to represent such a virtual machine or a bare-metal server in this
+document.
+
+`ExternalEntity` is used to represent the network interface defined in an
+`ExternalNode` object, when applying an Antrea NetworkPolicy to it.
+antrea-controller creates an `ExternalEntity` for each network interface when 
+it finds an `ExternalNode` is created, which can be used when applying Antrea
+NetworkPolicy.
+
+antrea-agent is running on the external Node which is described as `ExternalNode`,
```

Comment:
same: `antrea-agent`

Reply:
`antrea-agent` is running on the external Node and it controls network connectivity and security by